controllers/imagerepository_controller.go

```diff
@@ -210,6 +265,28 @@ func (r *ImageRepositoryReconciler) scan(ctx context.Context, imageRepo *imagev1
 		options = append(options, remote.WithAuth(auth))
 	}
 
+	if accountId, awsEcrRegion, _, _, _, err := parseAwsImageURL(imageRepo.Spec.Image); err == nil {
```

Comment:
Just a nit but since this is the only usage of the func, you could change the signature to only return what's needed here.

Reply:
I intended them for fine-grained matching against repository name and/or tag... but I removed them for now for code style.